library/Notifications/Widget/Schedule.php

```diff
@@ -118,7 +118,7 @@ public function assemble()
                 new Link(
                     [
                         new Icon('plus'),
-                        t('Add new entry')
+                        t('Add Entry')
```

Comment:
So the idea behind this is:

* You create/delete an Object (e.g. "New Schedule”), when you’re in the “schedules” views (Create)
* You add/remove “entry" objects to that “schedule", when you’re in the “schedules” view (Create and link)

This helps
* to clarify the different relations and
* with distinguishing the different actions better, especially in a view with many buttons in the same design

I decided against additionally highlighting the “add entry” button as a primary button, because it’s not that important in that view and would distract from the actual content which is the schedle itself.

![Screenshot 2023-05-23 at 13 27 30](https://github.com/Icinga/icinga-notifications-web/assets/6977434/767bf426-412e-4adb-b721-708545928412)

Reply:
Ah, okay, but I also *add* a new rule to the list of already existing event rules. I also *add* a new contact to the list of available contacts. In both cases, the relation is the list on the left. And to a list you *add* new items. The perfect label would be **Add New Contact**, but that's a bit redundant IMHO.

So if you think `Add Entry` is fine on the schedules view, I don't understand why you think `Add Contact` is not, on the contacts list.